lib/Serialization/Deserialization.cpp

```diff
@@ -5951,6 +5951,33 @@ void ModuleFile::loadAllMembers(Decl *container, uint64_t contextData) {
   }
 }
 
+static llvm::Error consumeErrorIfXRefNonLoadedModule(llvm::Error &&error) {
+    // Missing module errors are most likely caused by an
+    // implementation-only import hiding types and decls.
+    // rdar://problem/60291019
```

Comment:
I think it's okay to add a radar number into the commit message, but in the code they are not useful to any Swift developers outside of Apple. I would just try to say as much as possible about the problem in the comment itself (which you already did!) and move the radar link into a footnote in the git commit message.